packages/terra-framework-site/src/App.jsx

```diff
@@ -0,0 +1,180 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import Base from 'terra-base';
+import SlidePanel from 'terra-slide-panel';
+import ContentContainer from 'terra-content-container';
+import List from 'terra-list';
+import IconMenu from 'terra-icon/lib/icon/IconMenu';
+import ThemeProvider from 'terra-theme-provider';
+import CollapsibleMenuView from 'terra-collapsible-menu-view';
+import styles from './site.scss';
+
+const propTypes = {
+  children: PropTypes.node,
+};
+
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
+
+const themes = {
+  'Default Theme': '',
+  'Consumer Theme': 'cerner-consumer-theme',
```

Comment:
Do we want to include consumer theme ?

Reply:
we currently are using the 3 themes on the terra-core site so I think it's fine